Fabric.Identity.API/scripts/tests/readme.md

```diff
@@ -0,0 +1,16 @@
+# Running CatalystDosIdentity Functional tests
+
+## Setup Docker
+Docker needs to be up and running and set to run Linux containers.  
+
+The script run-functional-tests.sh starts up the docker enviroment but also runs functional tests which will pollute the enviroment for CatalystDosIdentity testing as well as shut down the identity container once finished testing. 
+
```

Comment:
Since `line 34` may change, should we reference the snippet of code to remove if it's not too large (or at least the first and last lines of the code block to ignore).